src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -129,6 +140,12 @@
           + " Use 0 for no timeout";
   public static final String SIMPLE_CONSUMER_POOL_TIMEOUT_MS_DEFAULT = "1000";
 
+  public static final String SIMPLE_CONSUMER_MAX_POLL_TIME_CONFIG = "simpleconsumer.max.poll.time";
```

Comment:
I tend to agree with @ewencp regarding the "simple consumer" nomenclature. With the new consumer, since there is only one consumer api, is it more appropriate to have one rest resource for all the new consumer features, rather than splitting into two separate resources one for manual partition assignment and group-based consumers.